wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -1,3 +1,7 @@
+# -*- coding: utf-8 -*-
+"""
+    odm2_timeseries_dao.py
+"""
```

Comment:
The docstring is not mandatory but rather a recommendation. We don't need to follow that to the letter if it does not make sense. I'd rather have no docstring than one that has only the filename. (I'll disable docstrings tests btw to avoid that. Humans are better than linters to decided when to add them :smile:)